embedded/store/ongoing_tx.go

```diff
@@ -34,9 +34,10 @@ type OngoingTx struct {
 }
 
 type EntrySpec struct {
-	Key      []byte
-	Metadata *KVMetadata
-	Value    []byte
+	Key         []byte
+	Metadata    *KVMetadata
+	Value       []byte
+	Constraints *KVConstraints
```

Comment:
I think Constraints should be handled at OngoingTx. Because it will be posible to unify MVCC validations for RW-transactions quite easily later on

Reply:
Moved to OngoingTx